static/js/documentation.js

```diff
@@ -6,7 +6,7 @@ $(function() {
   }
 
   // Set class 'active' for the appropriate subnav link
-  var pageUrl = location.href.match(/(.+\/millennium\/[^\/]+\/)/)[0]
+  var pageUrl = location.href.match(/(.+\/((millennium)|(soarian))\/[^\/]+\/)/)[0];
```

Comment:
do we need to capture mill/soarian now? or just allow either to match

Reply:
Just allow either to match. The soarian sidebar is messed up on the site because of this line.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/431/commits/0d02ee94fec4a24fa1ed9f0ddac0dde3c01b6dbd#diff-46bd8686b23677fb5c466a0f71fa66fdR9